bin/verify_rke2_charts

```diff
@@ -0,0 +1,111 @@
+#!/bin/sh
+
+set -e
+
+. libstd-ecm.sh
+. librancher-ecm.sh
+
+# Defaults
+REPOSITORY=rancher/rke2
+LOCAL_RKE2=''
+BRANCH='master'
+IGNORE_LIST=''
+
+usage() {
+    echo "usage: ${0} [hbilr]
```

Comment:
Can you include `-d`?

Reply:
Yeah, I intentionally skipped on this and the other scripts I have created, since I thought it was a developer only function. Let me update them.

Reply:
Renamed that flag to `-x` so we can continue using `-d` for dry run. I added the usage text as well.

Reply:
I love it. Thanks!